portal-registry/src/test/java/com/networknt/portal/registry/PortalRegistryTest.java

```diff
@@ -88,15 +88,23 @@ public void doRegisterAndAvailable() throws Exception {
         Assert.assertFalse(client.isRegistered(service2));
     }
 
+    @Ignore
```

Comment:
Since the subscribe related code was commented in https://github.com/networknt/light-4j/commit/f8b87acef5f9738dee2e611f184619ec46e507fb, the current code in this class is not related to `subAndUnsubService`. I suggest simply **ignore** this class for now.

Reply:
Yes. Agreed.

Reply:
the ignore is not imported

Reply:
Added in https://github.com/networknt/light-4j/pull/862. Thanks!